coordinator/src/main/scala/com/socrata/datacoordinator/service/collocation/Collocator.scala

```diff
@@ -6,14 +6,19 @@ import com.rojoma.json.v3.util._
 import com.socrata.datacoordinator.common.collocation.{CollocationLock, CollocationLockError, CollocationLockTimeout}
 import com.socrata.datacoordinator.id.DatasetInternalName
 import com.socrata.datacoordinator.resources.collocation.{CollocatedDatasetsResult, DatasetNotInStore, SecondaryMoveJobRequest, StoreNotAcceptingDatasets}
+import com.socrata.datacoordinator.service.collocation.CollocationRequest.CostLimits
```

Comment:
Instead of importing this here I would just fully qualify it on line 16. Everything else should already have access to the companion object.